app/src/main/res/values/strings.xml

```diff
@@ -91,6 +91,12 @@
 
 
  <!-- Add Activity -->
+    <!-- Activities -->
+    <string name="no_activities_scheduled">"No activities have been scheduled yet."</string>
+    <!-- Activity Item -->
+    <string name="location_text">Location</string>
+
+    <!-- Add Activity -->
```

Comment:
nit: I'm a bit confused about the comments (Add Activity appearing twice, for example). In case we use some strings in both the Add Activity and Activities screen, can we leave a single comment `<!-- Activities -->` to avoid confusion? Let me know what you think

Reply:
Thanks for pointing this out, this was a merge error. I'll leave only the `<!-- Activities --> ` comment.